automates/text_reading/src/test/scala/org/clulab/aske/automates/text/TestFunctionFragments.scala

```diff
@@ -0,0 +1,22 @@
+package org.clulab.aske.automates.text
+
+import edu.stanford.nlp.ie.machinereading.structure.EventMention
+import org.clulab.aske.automates.TestUtils._
+
+class TestFunctionFragments extends ExtractionTest {
+
+  val t1 = "In subsection 4a, the remaining large-scale parameter is the depth of the mixed layer."
+  failingTest should s"find function fragments from t1: ${t1}" taggedAs (Somebody) in {
+    val desired = Seq("depth of the mixed layer")
+    val mentions = extractMentions(t1)
+    testUnaryEvent(mentions, "Function", "input", desired)
```

Comment:
Ah, I see that the test is used. Then keep it and delete printouts. Also, try to break the test in some way (e.g., change spelling in desired and/or some action) to make sure you it is testing what you want it to.

Reply:
Oh right, I should delete the printouts. I'll also try breaking the tests. Thanks for the advice! :)